daemon/rpc_state.go

```diff
@@ -109,11 +113,14 @@ func statusStream(stateChan <-chan interface{},
 			case events.DataDisconnect:
 				if err := srv.Send(
 					&pb.AppState{State: &pb.AppState_ConnectionStatus{
-						ConnectionStatus: &pb.ConnectionStatus{
-							State:  pb.ConnectionState_DISCONNECTED,
-							ByUser: e.ByUser,
-						},
-					}}); err != nil {
+						ConnectionStatus: &pb.StatusResponse{
+							State:      pb.ConnectionState_DISCONNECTED,
+							ByUser:     true,
```

Comment:
I think this should be set to `e.ByUser`. For example in `libtelio.go` we can have the disconnected event caused by node disconnection or manual disconnection and `ByUser` is true only for manual disconnection